### DISCUSSION TITLE
Anything like a constellation / sky atlas overlay?

### DISCUSSION THREAD

**Discussion by tjzepf** [aaronwmorris/indi-allsky#1396] [General]
One feature I've missed in indi-allsky is the ability to overlay the current image with a map of constellation lines and well-known deep sky objects. I sometimes want to know if an astrophotography target is clearing my obstructions, and an overlay would help in that regard. Is there a solution for this already out there, or some other way to achieve that end?

**Comment by aaronwmorris** [comment]
I tried to integrate VirtualSky a couple of years back and I had a lot of trouble getting the VirtualSky canvas to sit on top of the indi-allsky image canvas.

At the time, TJ's allsky project displayed their images as an img element where I used a canvas which eliminates the flicker that occurs between images.  VirtualSky seems to be designed to allow transparency with an img background, but the canvas never worked right.

I may try it again eventually.  However, even the allsky project seems to recognize that VirtualSky appears to be an abandoned project.  I am not sure how wise it is to invest a lot of time in it.

**Comment by tjzepf** [comment]
Thanks for taking the time and the explanation. Good to know!

**Comment by bendenajones** [comment]
Stellarium with a 360 transparent bitmap of your surroundings is what I use beyond the astophotography tools that show the target relative to your lat/long/height for local horizon. 
This is what it looks like at night:
![image](https://github.com/user-attachments/assets/79b95b4b-4d15-45c4-afd1-955523291e30)
another  view with my background more visible
![image](https://github.com/user-attachments/assets/b06f437f-ca3b-4787-ad71-397bde5e1bc4)
